infra/gcp-broad/main.tf

```diff
@@ -527,15 +527,6 @@ module "ci_gsa_secret" {
   ]
 }
 
-resource "google_artifact_registry_repository_iam_member" "artifact_registry_viewer" {
-  provider = google-beta
```

Comment:
Was this just duplicated and not used?

Reply:
ya, I checked that it's the same as the other block. I guess terraform didn't have a problem with two role assignments that are the same and just treated it like a no-op.